src/main/java/com/programmers/application/domain/voucher/VoucherType.java

```diff
@@ -0,0 +1,5 @@
+package com.programmers.application.domain.voucher;
+
+public enum VoucherType {
+    FIXED, PERCENT
+}
```

Comment:
[nit] 
```suggestion
public enum VoucherType {
    FIXED, 
    PERCENT,
    ;
}
```
이렇게 하면 나중에 변경사항 발생했을 때 git에서 깔끔하게 보여서 좋아요 ㅎㅎ